tests/acceptance/shared/util/cluster.go

```diff
@@ -0,0 +1,317 @@
+package util
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	error2 "github.com/k3s-io/k3s/tests/acceptance/core/service/error"
+	"github.com/onsi/gomega"
+)
+
+type Node struct {
+	Name       string
+	Status     string
+	Roles      string
+	Version    string
+	InternalIP string
+	ExternalIP string
+}
+
+type Pod struct {
+	NameSpace string
+	Name      string
+	Ready     string
+	Status    string
+	Restarts  string
+	NodeIP    string
+	Node      string
+}
+
+// ParseNodes parses the nodes from the kubectl get nodes command
+// and returns a list of nodes
+func ParseNodes(print bool) ([]Node, error) {
+	nodes := make([]Node, 0, 10)
+	res, err := RunCommandHost(GetNodesWide + KubeConfigFile)
+	if err != nil {
+		return nil, err
+	}
+
+	nodeList := strings.TrimSpace(res)
+	split := strings.Split(nodeList, "\n")
+	for _, rec := range split {
+		if strings.TrimSpace(rec) != "" {
+			fields := strings.Fields(rec)
+			node := Node{
+				Name:       fields[0],
+				Status:     fields[1],
+				Roles:      fields[2],
+				Version:    fields[4],
+				InternalIP: fields[5],
+				ExternalIP: fields[6],
+			}
+			nodes = append(nodes, node)
+		}
+	}
+	if print {
+		fmt.Println(nodeList)
+	}
+
+	return nodes, nil
+}
+
+// ParsePods parses the pods from the kubectl get pods command
+// and returns a list of pods
+func ParsePods(print bool) ([]Pod, error) {
+	pods := make([]Pod, 0, 10)
+	podList := ""
+
+	res, _ := RunCommandHost(GetPodsWide + KubeConfigFile)
```

Comment:
Curious. Some other files seem to be utilizing util.KubeConfigFile. is this KubeConfigFile defined separately elsewhere? Maybe we should keep it uniform throughout?